### PR TITLE
Update projects.json Added my Project in it.

### PR DESCRIPTION
**Pull request by Hareesh108**
## Related Issue

data: project addition by Hareesh108 #933

## Description

I have added my project in it.

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -58,6 +58,24 @@
         "tech": ["React", "Express.js", "MUI", "Firebase", "Node.js"]
       }
     ]
+  },
+    {
+    "github_username": "Hareesh108",
+    "Social_media": {
+      "gitHub": "https://github.com/Hareesh108,
```
Comment (priyankarpal) [inline src/DB/projects.json:65]:
```suggestion
      "gitHub": "https://github.com/Hareesh108",
```

File: src/DB/projects.json [hunk 2]
```diff
@@ -58,6 +58,24 @@
...
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": ""
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Hareesh108/NotesTaker",
+        "title": "Personal Taker",
+        "description": "This web based note taker application",
+        "tech": ["React", "HTML", "CSS", "JavaScript]
```
Comment (priyankarpal) [inline src/DB/projects.json:76]:
```suggestion
        "tech": ["React", "HTML", "CSS", "JavaScript"]
```

**Comment by Hareesh108** [comment]
@priyankarpal Thanks.  i have updated changes.

**Review by priyankarpal — APPROVED** [review]
Thank you for contributing your projects here 👏🏻 🎉

**Pull request merged**